lib/perl/Genome/Model/ReferenceAlignment/Command/LibraryDuplicationRates.pm

```diff
@@ -0,0 +1,86 @@
+package Genome::Model::ReferenceAlignment::Command::LibraryDuplicationRates;
+
+use strict;
+use warnings;
+
+class Genome::Model::ReferenceAlignment::Command::LibraryDuplicationRates {
+    is => 'Command::V2',
+    has => [
+        builds => {
+            is_many => 1,
+            is => 'Genome::Model::Build::ReferenceAlignment',
+            shell_args_position => 1,
+            doc => 'List of builds to report on.',
+        },
+    ],
+};
+
+sub help_brief {
+    'A command to print Picard MarkDuplicates duplication metrics for a reference alignment build.'
+}
+
+sub help_synopsis {
+    'Dump reference alignment build duplication metrics to standard out in tab-delimited format.'
+}
+
+sub help_detail{
+    return <<"EOS"
+All reference alignment builds currently use Picard MarkDuplicates to flag PCR duplicates after merging the individual instrument data alignments.
+One output from Picard MarkDuplicates is a metrics file that provides a per library breakdown of PCR and optical duplicate reads.  This tool simply parses the metrics file and dumps those metrics along with basic build/model information in tab-delimited form.
```

Comment:
Now it solely relies on the db metrics.  Are you suggesting it should have support for both?  If the db metrics do NOT exist, call another subroutine that calls the old hash method?